wayland-display-core/src/comp/focus.rs

```diff
@@ -131,6 +134,69 @@ impl PointerTarget<super::State> for FocusTarget {
         }
     }
 
+    fn frame(&self, seat: &Seat<State>, data: &mut State) {
+        match self {
+            FocusTarget::Wayland(w) => w.frame(seat, data),
+            FocusTarget::Popup(p) => {todo!()},
```

Comment:
`p.wl_surface().frame()` expects `frame_number` which I'm not sure how to extract in here..

Reply:
It is probably defaulting to the wrong `frame`-method, try:
```suggestion
            FocusTarget::Popup(p) => PointerTarget::frame(&p.wl_surface(), seat, data),
```